tests/unit/test_utils.py

```diff
@@ -2,36 +2,21 @@
 import logging
 import unittest.mock as mock
 
-from jsonschema import ValidationError
-
-from broadway_api.utils.bootstrap import (
-    initialize_cluster_token,
-    initialize_course_tokens,
-)
+from broadway_api.utils.bootstrap import initialize_course_tokens
 from broadway_api.daos.course import CourseDao
 from tests.base import BaseTest
 
 logging.disable(logging.WARNING)
 
 
-class TestClusterTokenUtils(BaseTest):
-    def test_cluster_token_from_env(self):
-        with mock.patch.dict("os.environ", {"BROADWAY_TOKEN": "val"}):
-            self.assertEqual(initialize_cluster_token(), "val")
-
-    def test_cluster_token_generated(self):
```

Comment:
Maybe we re-add `initialize_cluster_token()` if we decide to make cluster token as an optional command-line argument. Then this can be re-added.